src/rain-report/rain-report.controller.spec.ts

```diff
@@ -30,7 +30,7 @@ describe('RainReportController', () => {
     it('should call `RainReportService.all()` to retrieve data', async () => {
       (mockRainReportService.all as jest.Mock).mockResolvedValue(SEED_DATA);
 
-      const response = await controller.index();
+      const response = await controller.index(USER_ID);
```

Comment:
Modified controller specs to pass in the `userId`. At runtime this is handled by the `@Headers()` decorator which is configured to return the `x-userid` header value.